docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,720 @@
+every week may not end up helping you as much as it would for our project, and you may encounter
+issues and concepts we won't be covering, however, feel free to use us as a resource for fixing
+and understanding them as they pop up.
+
+#### End Goal
+
+If you're watching the class online, here's where we'll show an example of the final project.
+
+### What technologies will I be learning?
+
+Really quickly, let's start off by getting some terminology out of the way. You'll be seeing 
+these terms really often over the rest of the course, and they're really important in 
+distinguishing the two main sides of the application. The frontend is the part that a user
+will see and interact with; it's the code that runs in a web browser in this case (like the Gmail 
+webpage), and it will usually be provided to every user. The backend is the part of the application that isn't often
+accessed by users, and the frontend will connect with the backend for performing calculations,
+storing data, and providing interactions between multiple frontend applications; the backend code
+is rarely provided to users, and is often held on a small set of servers. In the Gmail example, the
+backend would be the part of the application that deals with storing the actual email data, returning
+it to the frontend when requested, sending emails, and actually moving your email from inbox to
+trash, spam, archive, or wherever else.
+
+During this time, we'll be teaching and providing demonstrations for a Java backend 
+using a Vert.x framework (a dependency which handles setting up a lot of boilerplate stuff, or stuff which is
+commonly set up the same way) and Typescript frontend using a React framework with a PostgreSQL 
+database. 
+
+>Note: in the Important Links section of this document, we're providing access to the Jumpstart Docs webpage
+>which contains our 'Jumpstart Dictionary'. If you come across any terminology not explained here, be sure to check
+>that out!
+
+The frontend will use packages such as [Material-UI](https://material-ui.com/) for
+UI components, [Jest](https://jestjs.io/) for testing the frontend, 
+[Axios](https://github.com/axios/axios) as an HTTP client, and [React](https://reactjs.org/).
+
+The backend will use libraries such as [Vert.x and Vert.x-Web](https://vertx.io/),
+[PostgreSQL's](https://jdbc.postgresql.org/) database driver, 
+[Jackson](https://github.com/FasterXML/jackson)
+JSON utilities, [JUnit 5](https://junit.org/junit5/) for a testing framework, and
+[Mockito](https://site.mockito.org/) for a mocking framework.
+
+Throughout the course, we'll be covering a lot of topics like Git and terminal commands,
+HTML, CSS, and JavaScript, servers, HTTP and REST requests, testing, databases, APIs, and
+cloud deployment. 
+
+Don't worry if any of this seems unfamiliar or new to you, we'll be sure to cover all of this
+and answer any questions you may have.
+
+### How are the projects organized?
```

Comment:
For this whole section I would probably bold or underline the file / directory the first time you say it's name

Reply:
Should I do bullets too/instead?

Reply:
Also added it.